_spinnaker/armory_halyard.md

```diff
@@ -211,4 +212,88 @@ hal armory init [parameters]
  * `--path`: The path to where armory-install is already installed
 
 
+## hal armory secrets 
+
+Configure secrets management
+
+#### Usage
+```
+hal armory secrets [subcommands] [parameters]
+```
+
+#### Parameters
+ * `--deployment`: If supplied, use this Halyard deployment. This will _not_ create a new deployment.
+ * `--no-validate`: (*Default*: `false`) Skip validation.
+
+#### Subcommands
+ * `vault`: Configure secrets management with vault
+
 ---
+## hal armory secrets vault
+
+Configure secrets management with vault
```

Comment:
I'd mention it's for the services - not for Halyard itself and then point to a doc to let Halyard talk to Vault.